src/workers/postUpdated.ts

```diff
@@ -412,9 +417,17 @@ const fixData = async ({
     data,
   });
 
+  let keywords: string[] = [];
+  if (data?.extra?.keywords) {
+    keywords = keywords.concat(data.extra.keywords);
+  }
+  if (data?.extra?.keywords_native) {
+    keywords = keywords.concat(data.extra.keywords_native);
+  }
```

Comment:
We should not change the existing logic of the keywords. We should keep this business logic in yggdrasil

Reply:
That would be preferred to handle it in Yggdrasil.
Guess a change would need to be done there then?
cc @denisb0

Reply:
No change is required there actually :)
It already follows the right logic

Reply:
In the sample content I got from Denis, there's different keywords in `keywords` and `keywords_native`. And the way I interpret the type in [Yggdrasil](https://github.com/dailydotdev/yggdrasil/blob/befaa6c92e3e1dc598e6fc2cdb8368c50bbb83f8/services/content_publisher/types.go#L35-L36) those keywords are gathered from different places, and [never merged](https://github.com/dailydotdev/yggdrasil/blob/befaa6c92e3e1dc598e6fc2cdb8368c50bbb83f8/workflows/entry_engines/youtube/workflow.go#L341-L342)?

Or am I misunderstanding?

Reply:
That's right but we don't want to merge them also. Native ones can be quite a garbage to our experience

Reply:
Yap

Reply:
Removed